tests/Util/CliLoginHelperTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace ItkDev\OpenIdConnectBundle\Tests\Util;
+
+use ItkDev\OpenIdConnectBundle\Util\CliLoginHelper;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Cache\Adapter\FilesystemAdapter;
+use Symfony\Component\Uid\Uuid;
+use Symfony\Contracts\Cache\CacheInterface;
+
+class CliLoginHelperTest extends TestCase
+{
+    public function testEncodeAndDecode()
+    {
+        $cache = $this->createMock(CacheInterface::class);
```

Comment:
Consider using https://symfony.com/doc/current/components/cache/adapters/array_cache_adapter.html instead.
It may not be a mock, but its useful for testing purposes.